src/services/navigationBar.ts

```diff
@@ -660,7 +660,16 @@ namespace ts.NavigationBar {
         else if (isCallExpression(parent)) {
             const name = getCalledExpressionName(parent.expression);
             if (name !== undefined) {
-                const args = mapDefined(parent.arguments, a => isStringLiteralLike(a) ? a.getText(curSourceFile) : undefined).join(", ");
+                const args = mapDefined(parent.arguments, a => {
+                    if (isStringLiteralLike(a)) {
+                        const line = curSourceFile.getLineAndCharacterOfPosition(a.pos).line;

```

Comment:
Maybe there's a bunch of caching going on, but this looks more expensive than just linear searching for a line break.

Reply:
Line map is cached so this is not expensive